apps/inventory/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from utils_methods import *
+from utils_variables import *
+
+# Create your models here.
+class Warehouses(models.Model):
+    warehouse_id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=255,null=True,default=None)
+    code = models.CharField(max_length=255,null=True,default=None)
+    item_type_id = models.ForeignKey('masters.ProductItemType', on_delete=models.CASCADE, null=True, default=None, db_column='item_type_id')
+    customer_id = models.ForeignKey('customer.Customer', on_delete=models.CASCADE, null=True, default=None, db_column='customer_id')
+    address = models.CharField(max_length=255, null=True, default=None)
+    city_id = models.ForeignKey('masters.City', on_delete=models.CASCADE, null=True, default=None, db_column = 'city_id')
```

Comment:
state_id, country_id is missing here, pull the changes from main and check for the changes